front-end/src/components/Head.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import getNetwork from 'src/util/getNetwork';
+
+const Head = () => {
+	const network = getNetwork();
+	const title = `Polkassembly | ${network}`;
```

Comment:
We need to have the title of the post in there and the beginning of the description, if it has been set. Otherwise our SEO will still not be good because all pages will have the same title/desc.

Reply:
Rendering from context